render/package.json

```diff
@@ -8,9 +8,12 @@
 	"scripts": {
 		"build": "tsup",
 		"test": "DEV=true node dist/index.js",
+		"test:activity": "DEV=true node dist/activity.js",
```

Comment:
not confident this will be run by the `yarn test:render` from the top `package.json`